sfge/src/main/java/com/salesforce/config/UserFacingMessages.java

```diff
@@ -1,11 +1,32 @@
 package com.salesforce.config;
 
 /**
- * Contains error message constants that will be displayed to users. TODO: move all other
- * user-facing messages here.
+ * Contains error message constants that will be displayed to users.
+ * TODO: move all other user-facing messages here.
+ * TODO: Reorganize these messages into a layered system like in {@link com.salesforce.apex.jorje.ASTConstants}
  */
 public final class UserFacingMessages {
 
+    public static final class RuleDescriptions {
+        public static final String UNNECESSARILY_EXTENSIBLE_CLASS_RULE =
```

Comment:
Revisiting the rule name, how about something like, "UnextendedVirtualClassRule", to keep up with the "unused" theme?